lib/config/configuration.js

```diff
@@ -727,6 +727,8 @@ Configuration.prototype.registerDefaultRules = function() {
 
     this.registerRule(require('../rules/validate-newline-after-array-elements'));
 
+    this.registerRule(require('../rules/require-parentheses-around-arrow-param'));
```

Comment:
Might make sense to move es6 require rules in special helper, or at least in spacial place in this method